CHANGELOG.md

```diff
@@ -4,6 +4,38 @@
 
 <!-- next version -->
 
+## v1.0.0-rc10/vv0.76.0
```

Comment:
```suggestion
## v1.0.0-rc10/v0.76.0
```

Reply:
For some reason I can't commit this suggestion.

Reply:
Hmmm...i also can't submit the suggestion :(